addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -181,6 +182,16 @@ def _search_description(self, operator, value):
         allocations = self.sudo().search(domain)
         return [('id', 'in', allocations.ids)]
 
+    @api.depends('name', 'date_from', 'date_to')
+    def _compute_description_validity(self):
+        for allocation in self:
+            name_validity = "{} (from {} to ".format(allocation.name, allocation.date_from.strftime("%b %d %Y"))
```

Comment:
You need to use `_()` otherwise strings will not be translated.

You should also avoid strings concatenation with the `_()` as word orders will change based on the language :-)

Reply:
Done, thanks :)